content/blog/2020-11-25-november-20-community-gems.md

```diff
@@ -0,0 +1,195 @@
+---
+title: November '20 Community Gems
+date: 2020-11-25
+description: |
+  A roundup of technical Q&A's from the DVC community. This month, learn how
+  to clean your cache and use Git hooks with DVC. And here's an early holiday 
+  gift- new Bitbucket support for CML!
+descriptionLong: |
+  A roundup of technical Q&A's from the DVC community. This month, learn how
+  to clean your cache and use Git hooks with DVC. And here's an early holiday 
+  gift- new Bitbucket support for CML!
+picture: 2020-11-25/cover.png
+author: elle_obrien
+commentsUrl: https://discuss.dvc.org/t/october-20-community-gems/535
+tags:
+  - Discord
+  - Gems
+  - CML
+  - Cache
+  - Bitbucket
+---
+
+## DVC questions
+
+### [Q: If I make a new Git branch, how can I synchronize it with DVC?](https://discord.com/channels/485586884165107732/485596304961962003/773498570795778058)
+
+Generally, yes. When you make a new Git branch in your project:
```

Comment:
`Generally, yes.` reads a bit wrong here? (question is `how`)

Reply:
Noted, changed to "Here's what we recommend:"